deployer/templates/fluentd.yaml

```diff
@@ -343,4 +343,4 @@ parameters:
 -
   description: 'Fluentd buffer size limit'
```

Comment:
From my understanding, the environment variable BUFFER_SIZE_LIMIT is initially set by ansible. (Both fluentd and mux, defaults to 1m)
openshift_logging/defaults/main.yml:openshift_logging_fluentd_buffer_size_limit: 1m
openshift_logging_fluentd/templates/fluentd.j2:          value: "{{ openshift_logging_fluentd_buffer_size_limit }}"
openshift_logging_mux/defaults/main.yml:openshift_logging_mux_buffer_size_limit: 1m
openshift_logging_mux/templates/mux.j2:          value: "{{ openshift_logging_mux_buffer_size_limit }}"

Then, fluentd/run.sh checks BUFFER_SIZE_LIMIT again and if it's not set, reinitialize it to 1m and it's set to buffer_chunk_limit as well as used for calculating buffer_queue_limit.

I'm also wondering deployer/templates/fluentd.yaml is in-use or not...